opentech/apply/funds/templates/funds/includes/review_table_row.html

```diff
@@ -1,22 +1,39 @@
-{% for review in reviews %}
-    <tr class="tr--subchild {{ class }} {% if missing %}no-response{% endif %}">
-        {% if missing %}
-            <td class="reviews-sidebar__author" colspan="2"><span>{{ review }}</span></td>
-            <td>-</td>
-            <td>-</td>
-        {% else %}
-            <td class="reviews-sidebar__author" colspan="2">
-                {% if request.user.is_apply_staff or request.user == review.author %}
-                    <a href="{% url 'apply:submissions:reviews:review' submission_pk=review.submission.id pk=review.id %}">
-                        <span>{{ review.author }}</span>
-                    </a>
-                    <div class="reviews-sidebar__date">{{ review.updated_at|date:"Y-m-d" }}</div>
-                {% else %}
-                    <span>{{ review.author }}</span>
+{% load wagtailimages_tags %}
+
+<tr class="tr--subchild {{ class }} {% if missing %}no-response{% endif %}">
+    {% if missing %}
+        <td class="reviews-sidebar__author" colspan="2">
+            <span>
+                {{ reviewer }}
```

Comment:
Can you squash this into one line maybe? `{% if role %}{% image role.icon max-12x12 %}{% endif %}`

Reply:
Yes can do